contents/docs/feature-flags/early-access-feature-management.mdx

```diff
@@ -1,33 +1,54 @@
+We've prebuilt a site app that adds a user early access feature modal to your site. To use it:
 
-## Setup
+1. Ensure you set `opt_in_site_apps: true` in your PostHog initialization configuration.
 
-### Option 1
+2. Search for "[Early Access Features App](https://app.posthog.com/project/apps?name=Early%20Access)" in the "Browse Apps" tab.
 
-We've prebuilt [a site app](/tutorials/build-site-app) that gives you a widget out of the box. The site app just requires that you specify a `data-attr` that should be clicked on to trigger the widget to appear.
+3. Enable it by clicking the blue gear, setting a `data-attr` selector (or enabling "Show features button on the page"), enabling the toggle and pressing save.
 
-Any configured early access features will appear on this panel, which you can configure to make your app visible for users. 
+4. If you did set a `data-attr` selector, add a component with that selector into your app. For example, if you set your selector to `#beta-button`, you need to add an element like `<button id="beta-button">Public Betas</button>`.
+
+Once set up, early access features appear on this panel for users to opt in or out of.
 
 ![Create feature flags](../../images/features/feature-flags/early-access-feature-demo.png)
 
-### Option 2
+### Option 2: Custom implementation
+
+You can fully customize your early access management experience using `getEarlyAccessFeatures` and `updateEarlyAccessFeatureEnrollment` from PostHog's [JavaScript Web library](/docs/libraries/js#early-access-feature-management).
+
+- `getEarlyAccessFeatures` is called with a callback function that receives a list of early access features with their flag key, name, description, and documentation link. You can use this to build a UI for users to opt in or out of features.
+
+- `updateEarlyAccessFeatureEnrollment` is called with a feature flag key and a boolean value to opt in or out of the feature. This updates the user's opt-in status for the feature.
 
-Posthog-js 1.55 exposes `getEarlyAccessFeatures` and `updateEarlyAccessFeatureEnrollment`, which enable you to wrap your own experience around early access features. [API Docs](/docs/libraries/js#early-access-feature-management)
+Other useful functions for custom implementations are `isFeatureEnabled()` and `useActiveFeatureFlags()` (React only). These help you separate early access features users have opted into from those they have not.
 
-For a sample implementation, [see the site app code here](https://github.com/PostHog/early-access-features-app/blob/main/site.ts)
+For a sample implementation, see the [public beta tutorial here](/tutorials/public-beta-program#creating-a-beta-opt-in-page) or [the site app code here](https://github.com/PostHog/early-access-features-app/blob/main/site.ts).
```

Comment:
Let's include code examples of `getEarlyAccessFeatures`, `updateEarlyAccessFeatureEnrollment`, etc. and how to use them in this doc, instead of linking to the github or tutorial. It's easier to follow this way, and theres less jumping around

Reply:
Added